packages/unit-testing/stubs/ui/ui.js

```diff
@@ -0,0 +1,20 @@
+define(['./dialog/dialog', './message/message', './serverWidget/serverWidget'], function (dialog, message, serverWidget) {
+    /**
+     * SuiteScript UI Module (Client Side)
+     *
+     * @module N/ui
+     * @suiteScriptVersion 2.x
+     *
+     */
+    var ui = function() {};
+
+    ui.prototype.dialog = dialog;
+    ui.prototype.message = message;
+    ui.prototype.serverWidget = serverWidget;
+
+    /**
+     * @exports N/ui
+     * @namespace ui
+     */
+    return new ui();
```

Comment:
Adding this `ui` moudule makes me think if we should also add the `N` module at the root of our stubs. It's just an open question to consider later with the team.

Reply:
It made sense to include N/ui becuase of this Help Center section:
https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=section_156939471908.html
Which says:
Use [‘N/ui’] as the first argument of the define function as a shortcut to load the three modules (N/ui/dialog, N/ui/message, and N/ui/serverWidget) on server scripts, or the N/ui/dialog and N/ui/message on client scripts.

I would avoid creating N module as its usage is not recommended in SuiteScript. However, if we find similar examples in the doc for other sub-modules, we should expose them. E.g. N/crypto and N/crypto/certificate